### DISCUSSION TITLE
Is there a native LSQt downloadable ISO to boot a system? Debian LSQt appears difficult

### DISCUSSION THREAD

**Discussion by rajibando** [lxqt/lxqt#2615] [Q&A]
I searched the LSQt discussion group with the keywords:` lxqt iso bootable os`
There wasn't a discussion on the topic.

I have been successful in streaming audio from laptop to my bluetooth headset with the pavu bluetooth package and config.

I used the debian-live-1 2.6.0-amd64-lxqt.iso to install my system on laptop. The system installed from the ISO didn't have network-manager, dnsmasq and other associated packages installed.

However, blueman tray icon --> right click  --> Local Services directed that network-manager and dnsmasq be installed.

Having faced a problem tethering for internet with my smartphone's mobile/cellular ISP with Blueman and Connman, trying for two days with ChatGPT for a possible solution, I have hit a wall.

During the process I installed dnsmasq, network-manager, metwork-manager-gnome and other associated packages to try enabling Bluetooth for tethering, but failed.

ChatGPT, because human support is difficult to procure.

ChatGPT directed codes for terminal. 
`sudo lsof -i :53`
Based on output I provided, ChatGPT suggested that dnsmasq is clashes with connmand for DHCP and DNS, both using port 53.

Subsequently, we changed `/etc/dnsmasq.conf` with some lines to ensure that dnsmasg doesn't clash with connmand.
`port=1053`
`interface=pan1`
`listen-address=127.0.0.1,192.168.50.1`
`dhcp-range=192.168.50.10,192.168.50.50,12h`

In course of discussion, chatGPT said:

> Blueman includes NetworkManager because it's widely used for managing network connections, including Bluetooth. Many systems rely on NetworkManager for seamless integration with different types of networks, including Wi-Fi, Ethernet, and Bluetooth.
However, using NetworkManager alongside Blueman can create conflicts when both try to manage the same connection type (e.g., Bluetooth PAN). Blueman provides the option to use NetworkManager for users who prefer it or have a setup where it's the default.
 
Based on the discussion, we disabled and stopped Network-Manager.

Further, ChatGPT said that we Disable ConnMan's DNS and DHCP and allow dnsmasq to handle them instead. To try editing ConnMan's configuration to avoid conflicts:
        Create or edit /etc/connman/main.conf:
Adding the following lines:

> 
        [General]
        NetworkInterfaceBlacklist=eth*,wlan*,lo
        AllowDNSProxy=false

From the output of `sudo lsof -i :53` , connmand still appeared to grab the port.

Subsequently, we reached a stage when ChatGPT suggested that we inserted in systemctl file for connman:
`sudo systemctl edit connman` the lines:
>
        [Service]
        ExecStart=
        ExecStart=/usr/sbin/connmand --nodnsproxy

Then I checked with `sudo lsof -i :53`.

The above lines in `systemctl connman` file stopped connman from accessing the port 53, but upon repeated restarts I observed that `connman UI Setup` wasn't able to set up internet. So I subsequently silenced the lines. 

I can now access the internet like earlier. But internet through Bluetooth tethering yet remains to be accomplished.

**Comment by tsujan** [comment]
I wonder what's the relation to LXQt? You need to consult your distro docs/forums.

**Reply by rajibando** [reply]
Thank you for replying to my post, Mr. tsujan.

For me to satisfactorily answer your question, I would request that may please [the other post on the issue](https://github.com/lxqt/lxqt/discussions/2613) also be perused. Together, they would illustrate why I seek an independent LXQt iso from the LXQt Developers themselves rather than the debian iso with LXQt.

That other post indicates that the bluetooth internet tethering problem could be better solved with network-manager, whereas the debian LXQt iso I used installs `connman` instead of `netwiork-manager`. Which makes it difficult for me to seek solution from those who have the solution, as my LXQt set-up is different from the systems of those who already have a solution.

The complicated scenario resulted in my above post with my present Question posted as a Title.

**Reply by tsujan** [reply]
> I seek an independent LXQt iso from the LXQt Developers themselves

I'm afraid that won't happen. ISOs are specific to distros.

**Reply by stefonarch** [reply]
Once there was a LXQt comunity edition from Manjaro, maybe it will come back again. Making working iso's isn't an easy thing afaik. That said there is Lubuntu which provides a live medium and also a  Fedora spin:

https://lubuntu.me/downloads/
https://fedoraproject.org/spins/lxqt/download

**Reply by Bluey26** [reply]
If you have an internet connection meanwhile, and have some spare time, i would suggest you do to the following:

Make a lubuntu ISO bootable USB stick, then try to do the same steps that you did in the debian lxqt.

If i recall correctly Lubuntu comes with nm-tray by default. You could install blueman in the liveCD session and try to connect to internet via the "bluetooth thetering" 

If the above method works, it means you need to fix something in your Debian installed setup (like reinstalling nm-tray o conffiguring networkmanager properly).

If that does not work, you may need a newer version of blueman, which could be installed in debian testing, which is not easy to do either.

If those do not work either, maybe another thing is happening.

But all of this is not related to LXQt itself. nm-tray is bundled in most LXQt desktop enviroments shipped by the common Linux distributions, but it is not (as far as i know) an LXQt package. blueman is not either

**Reply by tsujan** [reply]
> Making working iso's isn't an easy thing

In Debian, it was a piece of cake a decade ago, when I was a Debian user. I made customized ISOs from my Debian system just for fun (a friend of mine used them for repairing MS Windows systems). But I don't remember how and don't know whether Debian still has this feature after a decade.

**Reply by Bluey26** [reply]
>  don't know whether Debian still has this feature after a decade.

I think that feature is still available (https://debian-live-config.readthedocs.io/en/latest/custom.html)

> If i recall correctly Lubuntu comes with nm-tray by default. You could install blueman in the liveCD session and try to connect to internet via the "bluetooth thetering"

Quoting myself, i just have tried this. After login-in in the Lubuntu 24.04 liveCD, i saw that blueman and nm-tray were installed by default, so i paired my bluetooth device(smartphone) and then i enabled "bluetooth tethering" in the phone. After this, opening nm-tray will show the smartphone connection and selecting it will give you internet.

Conclussion: there may be some missing step in your installed Debian setup, in the process from switching connman to networkmanager, or something else.

**Reply by tsujan** [reply]
> I think that feature is still available

Frankly, I've forgotten everything about Debian ;) I made live ISOs out of my installed Debian system, without downloading anything from Internet. I'd made a script (after looking into several scripts) and excluded those things that I didn't want to be in the ISO. Then I just used that script and waited for the ISO to be created (~30 minutes?).

When I came to Arch/Manjaro, I wasn't curious about ISO making because those Debian ISOs had no use to me.

EDIT: Found it! My script was based on this nice work: https://web.archive.org/web/20130423105647/http://www.remastersys.com/

**Reply by tsujan** [reply]
OK, this seems like a revival of `remastersys` but not limited to Debian-based systems: https://github.com/pieroproietti/penguins-eggs. Manjaro has packaged it.
